modules/worker-utils/src/lib/worker-utils/get-transfer-list.ts

```diff
@@ -72,7 +72,11 @@ export function getTransferListForWriter(object: object | null): object {
 
   Object.keys(clone).forEach((key) => {
     // Checking if it is an object and not a typed array.
-    if (typeof object[key] === 'object' && !ArrayBuffer.isView(object[key])) {
+    if (
+      typeof object[key] === 'object' &&
```

Comment:
Nit: This looks messy and I am not even sure the comment is completely accurate. 

FWIW I would extract this into a named function. I also note that this type of code tends to be surprisingly error prone, one can loose hours debugging to find out that this untested function sometimes makes the wrong choice.

I also note that loaders.gl already has a ton of type identifying helpers. https://github.com/visgl/loaders.gl/blob/4b87d91a8f746595cb7d3a7cea3b7e5eb97ca75e/modules/core/src/javascript-utils/is-type.ts. 

Since they are in code they may not be accessible here, but you could copy from there.